README.md

```diff
@@ -39,19 +39,69 @@ http://bender.astro.sunysb.edu/Castro/staging/Castro/html/
 
 ## Development Model:
 
-New features are committed to the `development` branch.  Nightly
-regression testing is used to ensure that no answers change (or if
-they do, that the changes were expected).  No changes should ever be
-pushed directly into `master`.
-
-On the first workday of each month, we perform a merge of
-`development` into `master`, in coordination with `AMReX`, `Maestro`,
-and `Microphysics`.  For this merge to take place, we need to be
-passing the regression tests.  To accommodate this need, we close the
-merge window into `development` a few days before the merge day.
-While the merge window is closed, only bug fixes should be pushed into
-`development`.  Once the merge from `development` -> `master` is done,
-the merge window reopens.
+Development generally follows the following ideas:
+
+  * New features are committed to the `development` branch.
+
+    Nightly regression testing is used to ensure that no answers
+    change (or if they do, that the changes were expected).
+
+    If a change is critical, we can cherry-pick the commit from
+    `development` to `master`.
+
+  * Contributions are welcomed from anyone.  *Any contributions that
+    have the potential to change answers should be done via pull
+    requests.*   A pull request should be generated from your fork of
+    Castro and target the `development` branch.  (If you mistakenly
+    target `master`, we can change it for you.)
+
+    Please add a line to `CHANGES` summarizing your change if it
+    is a bug fix or new feature.  Reference the PR or issue as
+    appropriate
+
+    If there are a number of small commits making up the PR, we may
+    wish to squash commits upon merge to have a clean history.
+    *Please ensure that your PR title and first post are descriptive,
```

Comment:
"First post" here means the PR description, correct? Also, this isn't a requirement of squashing...but I assume it is what comes up when you click on "Squash and merge" as the default?